### ISSUE TITLE
USWDS-Site - Documentation: Basic guidance on language in a11y page

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds-site#2517]
**guidance around language usage** - for components showing alternate options in differnent languagees (for example **Search** having a spanish variant), there should be info in the accessibility section reminding folks to set their page language in their html `<head>` or if they are only using a language different from the rest of the page in one area there is a `lang` attribute assigned programmatcally to that area

_Originally posted by @shkeating in https://github.com/uswds/uswds/discussions/5759#discussioncomment-8683887_

## Additional information

>**Use the HTML lang attributes** to set the language of the page (<html lang='en'>, for example). See [H57: Using the language attribute on the HTML element](https://www.w3.org/WAI/WCAG21/Techniques/html/H57) for more information.

[Two languages | U.S. Web Design System (USWDS)](https://designsystem.digital.gov/patterns/select-a-language/two-languages/#:~:text=If%20the%20text%20will,element%20for%20more%20information.)
